test/acceptance/test_cli.py

```diff
@@ -29,7 +30,7 @@ def assertReturnedStdoutEqual(self, expected_stdout, cmd):
 
     def test_exec_vint_with_valid_file_on_project_root(self):
         valid_file = str(Path('test', 'fixture', 'cli', 'valid1.vim'))
-        cmd = ['bin/vint', valid_file]
+        cmd = ['vint', valid_file]
```

Comment:
To make this clearer I think `assertReturnedStdoutEqual` should be passed `args` instead, and `vint` being removed from callers.
Otherwise it looks too much like `vint` is called.
(also below / with the rest of the patch)

Reply:
wfm.  sorry about the delay.